modules/network-flow-matrix.adoc

```diff
@@ -33,7 +33,7 @@ To view or download the complete raw CSV content for an environment, see the fol
 
 [NOTE]
 ====
-The network flow matrixes describe ingress traffic flows for a base {product-title} or {sno} installation. It does not describe network flows for additional components, such as optional Operators available from the Red Hat Marketplace. The matrixes do not apply for Hosted-Control-Plane, MicroShift, or standalone clusters.
+The network flow matrixes describe ingress traffic flows for a base {product-title} or {sno} installation. It does not describe network flows for additional components, such as optional Operators available from the Red Hat Marketplace. The matrixes do not apply for {hcp}, MicroShift, or standalone clusters.
```

Comment:
```suggestion
The network flow matrixes describe ingress traffic flows for a base {product-title} or {sno} installation. It does not describe network flows for additional components, such as optional Operators available from the Red Hat Marketplace. The matrixes do not apply for {hcp}, Red Hat build of MicroShift, or standalone clusters.
```
If this is other places that you've seen, let me know and I can port our attribute over to the common attributes file and update the OCP docs.

Reply:
Sure! thanks